src/Form/Button.jsx

```diff
@@ -9,11 +9,11 @@ const Button = ({ variant, sm, lg, rounded, disabled, block }) => {
         data-mdb-ripple-color="light"
         data-mdb-ripple="true"
         className={classNames(
-          variant === "secondary" && "bg-gray-600",
-          variant === "dark" && "bg-black ",
-          variant === "success" && "bg-green-600",
-          variant === "alert" && "bg-red-600",
-          variant === "warning" && "bg-yellow-600",
+          variant === "secondary" && "bg-gray-600 text-white",
+          variant === "dark" && "bg-black text-white",
+          variant === "success" && "bg-green-600 text-white",
+          variant === "alert" && "bg-red-600 text-white",
+          variant === "warning" && "bg-yellow-600 text-white",
```

Comment:
We should add some hover effects to buttons.

Reply:
Added ...